docs/Release-v2.1.0-Database-Migration.md

```diff
@@ -59,10 +59,11 @@ command. This option may be useful for instance to test out the
 migration to get a sense of the running time and increased MongoDB
 storage requirements for your typical datasets.
 
-1. Increase the storage capacity of your MongoDB to account for the
+1. Make sure that all users/runners have upgraded their SDKs!
```

Comment:
@allenleetc per offline discussion, may want to mention that any datasets that are edited by v2.1 SDKs will not be readable by < v2.1 SDKs until they are migrated up as well?